crates/matrix-sdk-ui/tests/integration/main.rs

```diff
@@ -82,22 +82,6 @@ async fn mock_context(
         .await;
 }
 
-/// Mocks the /event endpoint
-#[allow(clippy::too_many_arguments)] // clippy you've got such a fixed mindset
-async fn mock_event(
-    server: &MockServer,
-    room_id: &RoomId,
-    event_id: &EventId,
-    event: TimelineEvent,
-) {
-    Mock::given(method("GET"))
-        .and(path(format!("/_matrix/client/r0/rooms/{room_id}/event/{event_id}")))
-        .and(header("authorization", "Bearer 1234"))
-        .respond_with(ResponseTemplate::new(200).set_body_json(event.into_raw().json()))
-        .mount(server)
-        .await;
-}
-
```

Comment:
Removed this, as it's no longer used anywhere.

Reply:
Nice!